### ISSUE TITLE
Configurable tooltip

### ISSUE THREAD

**Issue by lipido** [lipido/kargos#4]
@trmdi suggested "It would be more flexible if the tooltip could be changed, for example users could hover over the time icon to see the date, no need to click to display the submenu, it is similar to the default Time date widget." #2

**Comment by lipido** [comment]
It is a very good idea. But I am not sure how to implement it. What about this:

```bash
#!/bin/bash
echo "title"
echo "tooltip text | tooltipmain=true"
echo "tooltip sub | tooltipsub=true"
echo "---"
echo "bla bla"
```
This will move the 'tooltip text' and 'tooltip sub' lines to the plasmoid tooltip. But... What about of **also** showing those lines in the plasmoid like regular lines? I think in an additional option like `tooltiponly=false` (default true, i.e. **move** the lines to the tooltip)

```bash
#!/bin/bash
echo "title"
echo "tooltip text | tooltipmain=true tooltiponly=false"
echo "tooltip sub | tooltipsub=true tooltiponly=false"
echo "---"
echo "bla bla"
```

**Comment by trmdi** [comment]
Maybe the tooltip is only needed when users hover mouse over the plasmoid.

**Comment by lipido** [comment]
Yes, of course. But, I'm talking about **what** is displayed on the tooltip when the mouse hovers the plasmoid. The line with `tooltipmain`/`tooltipsub` indicates the message, which will be only displayed in the tooltip unless the `tooltiponly=false` which, in such a case, it will be also displayed as a regular message in the bar.

**Comment by trmdi** [comment]
I also think like you:
```bash
#!/bin/bash
echo "title"
echo "tooltip text | position=tooltip-main,dropdown"
echo "tooltip sub | position=tooltip-sub"
echo "---"
echo "bla bla | position=tooltip-sub"
echo "another bla bla"
```
The default value of `postition` = dropdown.

**Comment by trmdi** [comment]
If this is implemented, RSS dropdown could be like that, very handy.
![img](https://github.com/cute-jumper/plasma-rss-indicator/raw/master/screenshots/list.png)

**Comment by lipido** [comment]
Cool. But the 'killer' feature of my kargos RSS (and the old KNewsTicker) is the continuous rotating news on the task bar! (no clicks to see news).

Regarding the tooltip feature. Ok, you mean that the tooltip can vary for each selected item (in bar or in dropdown). I would first implement all the BitBar "standard" features in order to reach 1.0 version as soon as possible. But I do not forget this feature!

**Comment by GM-Script-Writer-62850** [comment]
was this ever implemented in any form? i'd like to have a tooltip at the line 1 level
trying to do this:
![Screenshot_2021-11-15_08-58-44](https://user-images.githubusercontent.com/564653/141794630-f24dd8fe-5fb0-4d9f-a461-5e4b1d157888.png)
* this was taken on the xfce desktop using the genmon applet
